app/src/main/java/nerd/tuxmobil/fahrplan/congress/dataconverters/FetchScheduleResultExtensions.kt

```diff
@@ -5,14 +5,13 @@ import nerd.tuxmobil.fahrplan.congress.net.FetchScheduleResult as AppFetchSchedu
 
 fun NetworkFetchScheduleResult.toAppFetchScheduleResult() = AppFetchScheduleResult(
         httpStatus = httpStatus.toAppHttpStatus(),
-        eTag = eTag,
         hostName = hostName,
         exceptionMessage = exceptionMessage
 )
 
 fun AppFetchScheduleResult.toNetworkFetchScheduleResult() = NetworkFetchScheduleResult(
```

Comment:
Seems like this is only used in a test and can probably be removed.

Reply:
:heavy_check_mark: Removed in 2bc63f631809a584973a66f514c6be312b5dcdb5.